packages/node/example/bootstrappedApp/src/app.js

```diff
@@ -0,0 +1,43 @@
+const { Session } = require("../../../dist/Session");
+
+const argv = require("yargs/yargs")(process.argv.slice(2))
+  .describe("clientId", "The registered client ID.")
+  .alias("id", "clientId")
+  .describe("clientSecret", "The secret associated to the client ID.")
+  .alias("secret", "clientSecret")
+  .describe("refreshToken", "The API token (OAuth refresh token).")
+  .alias("token", "refreshToken")
+  .describe("oidcIssuer", "The OIDC issuer that issued the token.")
+  .alias("oidcIssuer", "issuer")
+  .describe("resource", "The resource to fetch")
+  .demandOption([
+    "clientId",
+    "clientSecret",
+    "refreshToken",
+    "oidcIssuer",
+    "resource",
+  ])
+  .locale("en")
```

Comment:
I don't think this is correct really - I just meant when you're demo'ing to the company you could switch your machine's locale to English, but not force this in the code itself...?

Reply:
Since all the descriptions are hard-coded in English, this keeps the help message consistent. If we were to support multiple locales in this script, then I'd agree with you, but I don't think that's the case here.